docs/gettingstarted.rst

```diff
@@ -1,16 +1,149 @@
+.. _gettingstarted:
+
 Getting started
 ===============
 
-Launch workflows
+About
+-----
+
+This tutorial explains how to install a local REANA cloud to get started with
+REANA development.
+
+Install minikube
 ----------------
 
-Launch Yadage workflows
-~~~~~~~~~~~~~~~~~~~~~~~
+REANA cloud uses `Kubernetes <https://kubernetes.io/>`_ container orchestration
+system. The best way to try it out locally is to set up `Minikube
+<https://kubernetes.io/docs/getting-started-guides/minikube/>`_. How to do this
+depends on your operating system. For example, on Arch Linux, you can install
+the following packages:
+
+- `virtualbox <https://www.archlinux.org/packages/community/x86_64/virtualbox/>`_
+- `virtualbox-guest-iso <https://www.archlinux.org/packages/community/x86_64/virtualbox-guest-iso/>`_
+- `virtualbox-host-modules-arch <https://www.archlinux.org/packages/community/x86_64/virtualbox-host-modules-arch/>`_
+- `minikube <https://aur.archlinux.org/packages/minikube/>`_ from AUR
+- `kubectl-bin <https://aur.archlinux.org/packages/kubectl-bin/>`_ from AUR
+
```

Comment:
We should add `docker` client as a requirement (open issue https://github.com/reanahub/reana/issues/18)